User_Reg_model_entryPoint/src/models/UserRegModel.java

```diff
@@ -0,0 +1,72 @@
+package models;
+import java.lang.reflect.Array;
+import java.util.*;
+
+//This class provides a constructor that provides attributes expected of a user 
+class User{
+    String username;
+    String firstname;
+    String lastname;
+    Date dob;
+
```

Comment:
- Make fields private to enforce encapsulation.
- toString(), equals(), and hashCode() Methods: Override these methods for better object representation and comparison.
- Use better variable naming, foinstance dateOfBirth is more understandable to dob